db/fixtures/en_de.rb

```diff
@@ -0,0 +1,39 @@
+require 'nokogiri'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.